judge/views/organization.py

```diff
@@ -681,6 +654,11 @@ def get_initial(self):
         initial['key'] = ''.join(x for x in self.organization.slug.lower() if x.isalpha()) + '_'
         return initial
 
+    def get_form_kwargs(self):
+        kwargs = super().get_form_kwargs()
+        kwargs['org_pk'] = self.organization.pk
```

Comment:
Because the new `AdminOrganizationMixin` is not add the `org_pk` to the form like the old `CustomAdminOrganizationMixin`, so i add it here